packages/test-framework/src/react-testing-library-support/CheckoutPageNodeObject.ts

```diff
@@ -164,14 +169,22 @@ export class CheckoutPageNodeObject {
                 );
                 break;
 
-            case CheckoutPreset.CheckoutWithMultiShipping:
+            case CheckoutPreset.CheckoutWithMultiShippingCart:
```

Comment:
The mock does not have `consignments` value, so it is more accurate to be called `CheckoutWithMultiShippingCart`.